build/openwisp_dashboard/module_settings.py

```diff
@@ -1,5 +1,6 @@
 import os
 from openwisp.utils import request_scheme
+from openwisp.settings import MIDDLEWARE
```

Comment:
```suggestion
from django.conf import settings
```

I think this can be replaced with this.
P.S: I didn't tried this.